pkg/skaffold/survey/survey.go

```diff
@@ -17,28 +17,38 @@ limitations under the License.
 package survey
 
 import (
+	"fmt"
 	"io"
 	"os"
-
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/color"
 )
 
 const (
 	Prompt = `Help improve Skaffold! Take a 10 seconds anonymous survey by running
    $skaffold survey`
+
+	Form = `Thank you for offering your feedback on Skaffold! Understanding your experiences and opinions helps us make Skaffold better for you and other users.
+   Our survey can be found here: https://forms.gle/BMTbGQXLWSdn7vEs6
```

Comment:
So this means we introduce yet another step for the user to do the survey. 
  - they see prompt
  - they invoke `skaffold survey`
  - now they copy into their web browser

Doesn't it make sense to open the survey link directly when running `skaffold survey`?  [github.com/pkg/browser](https://godoc.org/github.com/pkg/browser#OpenURL) looks good?

Or why not show the URL in the initial `Prompt`? (`Take a 10-second anonymous survey at https://forms.gle/BMTbGQXLWSdn7vEs6`).

And could we instead install a redirect at `skaffold.dev/survey`?

Reply:
i think it makes sense to just show the survey here and not add another step. When this was designed, we were going to show the survey questions, parse the responses and send using clearcut responses. 
Since we don't see this happenning any more in near future, i will drop this PR and just show the survey link here.

Reply:
I think it would be great to automatically open this link in a browser. I really hate copying things out of my terminal since my normal `cmd + C` shortcut doesn't work.

Reply:
done.

Reply:
Sorry, i forgot to update the resolution on this comment. Fixed it on the latest branch